src/pages/TestDrive/TestDrive.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Container, Form, Card } from "react-bootstrap";
+import { useLocation, useParams, Link, useNavigate } from "react-router-dom";
+import React, { useState } from "react";
+import "./TestDrive.scss";
+function TestDrive() {
+  let navigate = useNavigate();
+  const [mobileValid, setMobileValid] = useState(false);
+  const [emailValid,setEmailValid] = useState(false);
+  var pattern = /^(\+\d{1,3}[- ]?)?\d{10}$/;
+  let emailPattern = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+  const [validated, setValidated] = useState(false);
+
+  const handleSubmit = (event: any) => {
+    const form = event.currentTarget;
+    event.preventDefault();
+    event.stopPropagation();
+    if (form.checkValidity() === true) {
+      console.log("True")
+    }
+    setValidated(true);
+  };
+
+  const handleChange = (event: any) => {
+    const {name,value} = event.target
+    if(name==="contact"){
+        if (!(!value || pattern.test(value) === false)) {
+            setMobileValid(value);
+          }
+    }
+    else if(name==="email"){
+        if (!(!value || emailPattern.test(value) === false)) {
+            setEmailValid(true);
+          }
+    }
+  };
+
+  return (
+    <>
+      <Container className="testDrive--main--container">
+        <h5>Test Drive Form</h5>
+        <Card>
+          <Container>
+            <Form noValidate validated={validated} onSubmit={handleSubmit}>
+              <Form.Group className="mb-3" controlId="formGroupName">
+                <Form.Label>Name</Form.Label>
+                <Form.Control
+                  type="text"
+                  placeholder="Enter name..."
+                  required
+                  name="name"
+                />
+                <Form.Control.Feedback type="valid">
+                  Looks good!
+                </Form.Control.Feedback>
+                <Form.Control.Feedback type="invalid">
+                  Name Required
+                </Form.Control.Feedback>
+              </Form.Group>
+              <Form.Group className="mb-3" controlId="formGroupName">
+                <Form.Label>Mobile Number</Form.Label>
+                <Form.Control
+                  type="text"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel

Reply:
Changed the form input type for all